### PR TITLE
feat: Add a description

### LINKED ISSUE #2: test issue
glub glub

### LINKED ISSUE #3: Add a description
Please add a demo for this project.

### PR DESCRIPTION
**Pull request by sjiang1**
This is a demo pull request to resolve issue #3.

### CODE REVIEW

**Comment by Nick-Garska** [comment]
is this a review or a comment?

**Review by Nick-Garska — DISMISSED** [review]
nothing gets broken by this change.

**Review by jjjjess — DISMISSED** [review]
Here is my review!

**Review by amoots-emich — DISMISSED** [review]
LGTM

**Review by camdenf101 — DISMISSED** [review]
Looks good.

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# cosc481demotest
+Add a description
```
Comment (sjiang1) [inline README.md:1]: There is a typo here. You should fix that!

**Review by SoloTHatt — APPROVED** [review]
Looks Good

**Review by Desh776 — APPROVED** [review]
right

**Pull request merged**